url-template/build.boot

```diff
@@ -0,0 +1,29 @@
+(set-env!
+ :resource-paths #{"resources"}
+ :dependencies '[[cljsjs/boot-cljsjs "0.7.0"  :scope "test"]])
+
+(require '[cljsjs.boot-cljsjs.packaging :refer :all])
+
+(def +version+ "2.0.8")
```

Comment:
You need to separate lib and build-identifier to two vars: https://github.com/cljsjs/packages/wiki/Creating-Packages#51-buildboot

Reply:
Oops, fixed.